ingestion/tests/unit/test_usage_filter.py

```diff
@@ -210,26 +148,6 @@ class UsageQueryFilterTests(TestCase):
     Usage filter tests for database and schema filters
     """
 
-    @patch.object(OpenMetadata, "list_all_entities", mock_list_entities)
-    def test_prepare_mssql(self):
-        config = OpenMetadataWorkflowConfig.parse_obj(mock_mssql_config)
-        mssql_source = MssqlUsageSource.create(
-            mock_mssql_config["source"],
-            config.workflowConfig.openMetadataServerConfig,
-        )
-        mssql_source.prepare()
-        assert mssql_source.filters == EXPECTED_MSSQL_FILTER
-
-    @patch.object(OpenMetadata, "list_all_entities", mock_list_entities)
-    def test_prepare_redshift(self):
-        config = OpenMetadataWorkflowConfig.parse_obj(mock_redshift_config)
-        redshift_source = RedshiftUsageSource.create(
-            mock_redshift_config["source"],
-            config.workflowConfig.openMetadataServerConfig,
-        )
-        redshift_source.prepare()
-        assert redshift_source.db_filters == EXPECTED_REDSHIFT_FILTER
-
     @patch.object(OpenMetadata, "list_all_entities", mock_list_entities)
     def test_prepare_clickhouse(self):
```

Comment:
clickhouse stores the list of dbs involved in the query but rest of them does not so when a query contains table from different dbs like `db.schema.table` rest of the sources only stores the db on which the query was executed hence it filter outs the queries which might be relevant!